### PR TITLE
feat

### PR DESCRIPTION
**Pull request by ckahub**
#Signup
chrome-capture-2022-2-26 (4)

### CODE REVIEW

File: pages/signup.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                </ul>
+            </nav>
+              <!-----nav2-end----->
+
+             
+        </header>
+
+
+            <!---Signup form start--->
+
```
Comment (manishsom20) [inline pages/signup.html:61]: Remove extra spaces

File: style/signup.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
+
+:root{
+    --primary-color:black;
+    --secondary-color:tan;
+    --active-color:aqua;
+}
```
Comment (manishsom20) [inline style/signup.css:6]: use hex code for colors

File: style/signup.css [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    height: 1rem;
+    line-height: 1rem;
+    justify-content:space-evenly;
+    align-items:center;
+    box-shadow: none;
+    text-align: center;
+}
+.nav2 ul{
+    list-style: none;
+    /* text-align: center; */
```
Comment (manishsom20) [inline style/signup.css:34]: remove comented codes

**Review by manishsom20 — CHANGES_REQUESTED** [review]
CSS code structure could be improved. rest looks good to me

File: style/signup.css (hunk 3 above)
Comment (ckahub) [reply style/signup.css:34]: removed commented

File: style/signup.css (hunk 2 above)
Comment (ckahub) [reply style/signup.css:6]: yes

File: pages/signup.html (hunk 1 above)
Comment (ckahub) [reply pages/signup.html:61]: removed extra space

**Review by dusariudaykumar — APPROVED** [review]
Looks good to merge.

**Comment by ckahub** [comment]
thanx uday and manish for reviews

**Pull request merged**